extensions/amp-accordion/validator-amp-accordion.protoascii

```diff
@@ -81,6 +94,10 @@ tags: {  # <amp-accordion> > <section>
   attrs: {
     name: "[data-expand]"
   }
+  # amp-bind
+  attrs: {
+    name: "[expanded]"
```

Comment:
Will this break 0.1 if people try to bind to this value? Please do some manual testing using this on 0.1 to confirm nothing tricky happens. Otherwise we may need b/150399834 sooner, FYI @honeybadgerdontcare.

Reply:
It psuedo works in `0.1` if people bind to the `expanded` value.  Basically the `expanded` attribute causes the section to open or close so binding to it actually opens and closes the section.  It slightly wonky if animations are enabled since it will not animate open or close, it's just triggering the whether or not the section is displayed open or close.

I think it's a relatively low risk since nothing is breaking and binding to `expanded` in `0.1` is technical not supported behavior.  Let me know if you think otherwise.

Reply:
If it pseudo works and it's valid, people may start using it and expecting us to support it. The validator is as close we have to law about a commitment to support features. If we keep this as is for 0.1 we should modify 0.1 to throw errors if the `expanded` attribute gets mutated, as that should not be something people start relying on in that version.

Reply:
Great, I've added the error msg to `0.1` here: https://github.com/ampproject/amphtml/pull/33828